samples/rest/count_tokens.sh

```diff
@@ -170,4 +175,126 @@ curl "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash:c
           {"file_data":{"mime_type": "video/mp4", "file_uri": '$file_uri'}}]
         }]
        }'
-# [END tokens_multimodal_video_audio_file_api]
\ No newline at end of file
+# [END tokens_multimodal_video_audio_file_api]
+
+echo "[START tokens_cached_content]"
+# [START tokens_cached_content]
+wget https://storage.googleapis.com/generativeai-downloads/data/a11.txt
+echo '{
```

Comment:
nit: why echo to request.json instead if just putting this in the -d argument like all the other body json?

Reply:
Does the base64 break it?